### PR TITLE
Release 0.3-1

### PR DESCRIPTION
**Pull request by Rulox**
### Proposed changes
* Removed Ubuntu 14 from targets (not supported since April)
* New release notes

### CODE REVIEW

**Comment by Dean-Coakley** [comment]
Maybe out of scope for this PR but: 
Do you think it might be worth including a compatibility note of `NGINX Plus R14 or newer` in the README? - Or do you think a note at the end of release notes is sufficiently clear?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+* Create separate documentation for the configuration for different cloud providers: [aws](examples/aws.md) and [azure](examples/azure.md).
+
+UPGRADE:
+
+The upgrade process requires change the configuration file name. Below are the recommended steps to follow:
+1. Change the name of the configuration file from `/etc/nginx/aws.yaml` to `/etc/nginx/config.yaml`.
+2. Download the Release 0.3 nginx-asg-sync package for your OS and upgrade the package using the OS tools (dpkg or rpm).
+3. Check the logs of nginx-asg-sync to make sure that it is working properly after the upgrade.
+
+Note: the supported versions of NGINX Plus are R14 and higher.
```
Comment (pleshakov) [inline CHANGELOG.md]: R18 (because the client uses version 4)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
+## 0.3-1 (September 4, 2019)
+
+IMPROVEMENTS:
+
+* Add support for Azure Virtual Machine Scale Sets https://github.com/nginxinc/nginx-asg-sync/pull/24
+* Create separate documentation for the configuration for different cloud providers: [aws](examples/aws.md) and [azure](examples/azure.md).
+
+UPGRADE:
+
+The upgrade process requires change the configuration file name. Below are the recommended steps to follow:
```
Comment (pleshakov) [inline CHANGELOG.md]:
```suggestion
The upgrade process requires changing the configuration file name. Below are the recommended steps to follow:
```

File: CHANGELOG.md (hunk 1 above)
Comment (pleshakov) [reply CHANGELOG.md]: I would also add that "Ubuntu 14.04 (Trusty) is no longer supported"

**Pull request merged**